CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Fix(Accordion): header strips spaces when html tags are used
 - Fixed missing onDataClick handler for Table component in mobile view (#214)
 - Add Material Icons: 'Build', 'ViewCarousel', 'MailOutline', 'LabelOutline', 'Unarchive'
+- Fix TagBuilder not deleting tags properly with backspace (#223)
```

Comment:
0.8.3 is already on Master, could you please change it to 0.8.4 and update the changelog? Thanks.